biome.json

```diff
@@ -7,7 +7,13 @@
   },
   "files": {
     "ignoreUnknown": false,
-    "ignore": ["data/test", "dist"]
+    "ignore": [
+      "**/data/test",
+      "**/custom-data",
+      "**/dist",
+      "**/.tmp",
+      "**/__snapshots__"
+    ]
```

Comment:
The `coverage` folder also has to be excluded.

While trying things out, I have noticed that the `ci` script does not seem to work as expected. biome is reporting errors (because of formatting errors in coverage folder), but the script does not return with a non-zero exit code. You can try this by running `pnpm run ci` twice in `vitest-file-snapshots` for example.

I think the issue is that a single `&` is used in the `ci` script. This results in a background execution of parts of the script, which was the intention I guess. But the result of the biome check is ignored in the end, which is an issue.

Not sure if this is the case in all shells, though. I am using zsh. A simple test for me is to run `false & true` in the shell, which has a zero exit code.

Reply:
Thanks, the `coverage` directory is now excluded.

Good catch regarding the `ci` scripts. The singular `&` was not intentional, all sub scripts should be sequentialized. I fixed the scripts in 397cc9cf5e0e16e895000daecd752639e298a753.

In the future, we might introduce a build system into the repo to simplify task execution and enable parallelization and caching. But for the start, I wanted to avoid this complexity to focus on the features.